unstructured/staging/base.py

```diff
@@ -337,3 +338,92 @@ def filter_element_types(
         return filtered_elements
 
     return elements
+
+
+def convert_to_coco(
+    elements: List[Element],
+    dataset_description: str = f"Unstructured COCO Dataset {datetime.now().strftime('%Y-%m-%d')}",
```

Comment:
If we set the default dynamically in the signature like this it will be set once at loadtime, but I think the intent is to set it whenever the function is called. If you want to timestamp it to when the function is called, make the type `Optional[str]`, set the default to `None`, and then in the body of the function, when `dataset_description is None` set it to the timestamped value.